m4_sessions/1706-inning/group_one.md

```diff
@@ -1,19 +1,35 @@
 # Group One
 
 ### Session Title: Looking Good: the importance of nonverbal communication
+#### Session Description:
+We communicate all the time without saying a word, or even thinking about it! Learn about how nonverbal communication can make us better team members, improve our success in the workplace, and make us happier healthier developers :D
 
 ### Session Outcomes: 
 - Awareness of your own body language and how it affects others
-- Awareness of others' body language and how you respond
-- Trivia - tips and tricks
-- What some of the success you might have, losses you might suffer with body language
+- Awareness of others' body language and how you respond to it
+- Tips and tricks for awareness and empathy
+- Understand how nonverbal communication helps make you a stronger member of a team/Turing/workplace/humanity
 
 ### Session Outline:
-1. 10 minutes: mingling
-2. 15 minutes: activity
-3. 15 minutes: content (video/quiz/tips/tricks)
-4. 5 minutes: talk/listen - try to tell a story using one word. Listener: did you get it? Switch!
-5. 15 minute: discuss with your group for 15 minutes (3 questions)
-6. 10 minute: group wrapup/takeaways
+1. 5 minutes (Laura): mingling/intros
+2. 5 minutes (Laura): Group quiz - [Quiz Link](https://testyourself.psychtests.com/testid/3764)
+3. 5 minutes (Laura): Group quiz - [Quiz Link](https://www.theguardian.com/small-business-network/2016/mar/04/quiz-fluent-body-language-business-success-relationships)
+4. 10 minutes (Jonathan): Body language and professionalism - practice power poses, smiling, placebo effect, subconscious
+5. 15 minutes (Stephanie): Watch this video (sound off, pay attention to the body language, write impressions on the white board) - [Video Link](https://www.youtube.com/watch?v=_ZBKX-6Gz6A)
+6. 10 minutes (Leta): Turn and talk - try to tell the story of your commute to Turing this morning using one word (think "I am Groot"). Listener: did you get it? What emotions did you pick up on? Talk about what you were trying to communicate, how you interpreted it. Then switch!
+8. 15 minutes (Nick): Group wrapup/takeaways -
+    - Journal for a few minutes
+    - These aren't conscious actions: practice being more aware of your body language
+    - How can we support each other? Help each other smile, give nonverbal feedback, etc
+    - What's your responsibility during projects? Think about how this applies to your time at Turing/out in the workplace/in life
+9. 5 minute: Rewatch the video (sound ON) - [Video Link](https://www.youtube.com/watch?v=_ZBKX-6Gz6A)
```

Comment:
I think it would be great to have them go back into their pair share if they have time to share their reflections.